pyproject.toml

```diff
@@ -32,6 +32,8 @@ classifiers = [
 dependencies = [
     "openvino_tokenizers~=2025.1.0.0.dev"
 ]
+[project.optional-dependencies]
+testing = ["pytest>=6.0"]
```

Comment:
we have tests requirements here https://github.com/openvinotoolkit/openvino.genai/blob/master/tests/python_tests/requirements.txt